src/components/Post/Comments.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
If not using React object explicitly, remove its import as it has no impact on this file